vermouth/tests/molecule_strategies.py

```diff
@@ -225,13 +226,14 @@ def random_link(draw):
     -------
     hypothesis.searchstrategy.lazy.LazyStrategy
     """
-    link = draw(random_block(block_class=Link))
+    link = draw(random_block(block_class=Link,
+                             max_nodes=max_nodes, max_meta=max_meta))
     link.removed_interactions = draw(interaction_collection(
         link, interaction_class=DeleteInteraction, attrs=True,
     ))
-    link.molecule_meta = draw(attribute_dict())
+    link.molecule_meta = draw(attribute_dict(max_size=4))
```

Comment:
Why not `max_meta` here? It's unclear to me what the arguments to this functions (are supposed to) do.

Reply:
`max_meta` sets the maximum size of `(molecule|block|link).meta`. `link.molecule_meta` is something else: it is a filter on the `meta` of the molecule the link is applied to.